micro-demos/daemon_sets/demo.sh

```diff
@@ -33,6 +33,6 @@ run "kubectl --namespace=demos create -f $(relative daemon.yaml) --validate=fals
 run "kubectl --namespace=demos describe ds daemons-demo"
 
 tmux new -d -s my-session \
-    "$(dirname ${BASH_SOURCE})/split1_lhs.sh" \; \
-    split-window -h -d "sleep 10; $(dirname $BASH_SOURCE)/split1_rhs.sh" \; \
+    "sleep 10; $(dirname ${BASH_SOURCE})/split1_color_nodes.sh" \; \
```

Comment:
In case you don't already know, you can use  `tmux select-pane -t` to select the pane to focus on (e.g., https://github.com/kubernetes/contrib/blob/master/micro-demos/deployments/demo.sh#L35)